codecarbon/cli/main.py

```diff
@@ -20,6 +23,10 @@
 from codecarbon.core.schemas import ExperimentCreate, OrganizationCreate, ProjectCreate
 from codecarbon.emissions_tracker import EmissionsTracker
 
+AUTH_CLIENT_ID = "pkqh9CiOkp4MkPqRqM_k8Xc3mwBRpojS3RayIk1i5Pg"
```

Comment:
Isn't it dangerous to put this code in public ?

Reply:
The secret yes (used in the backend on /auth/login), the id no. It's a public information
That being said we still have some work to streamlmine and secure the auth process.

Reply:
This is a specific client for public usage of the authentication system.
This client ID is not secret, as it is only used to log in the user and after the user credentials will be stored in the credentials.json file, and use a different authentication workflow.
So it should be okay (and also is the recommended way to authenticate CLIs from the fief documentation : https://docs.fief.dev/integrate/python/cli/)